src/Adapter/EntityMapper.php

```diff
@@ -95,8 +95,7 @@ public function load($id, $id_lang, $entity, $entity_defs, $id_shop, $should_cac
                 }
                 $entity->id = (int) $id;
                 foreach ($object_datas as $key => $value) {
-                    if (array_key_exists($key, $entity_defs['fields'])
-                        || array_key_exists($key, $entity)) {
+                    if (isset($entity_defs['fields'][$key]) || isset($entity, $key)) {
```

Comment:
same here, we want null values

Reply:
Yeah yeah, I introduce back my uses of  property_exists and ... time to go.

WIP label added.